src/prebundle.ts

```diff
@@ -157,6 +160,18 @@ function emitPackageJson(task: ParsedTask) {
 
   pickedPackageJson.types = 'index.d.ts';
 
+  pickedPackageJson.type = 'commonjs';
+
+  if (assets['package.json']) {
+    try {
+      Object.assign(
+        pickedPackageJson,
+        // will be `{"type":"module"}` if the output is of esm format
+        JSON.parse(assets['package.json'].source),
```

Comment:
All properties in `source` will override `pickedPackageJson`, such as `types`, is this expected?

I prefer to only override the `type` property

Reply:
DONE.

ncc does not intend to produce packages or types at this time. It will only create a `package.json` with the `type` field if necessary, along with fields such as `exports` and `imports` in the foreseeable future

Reply:
get 👌